catalog/task/create-internal-request/0.1/create-internal-request.yaml

```diff
@@ -59,3 +67,34 @@ spec:
             addArches: `printf "%q" '$(params.addArches)'`
         YAML
         kubectl create -f ${resourceRequest}
+    - name: watch-internal-request-status
+      image: quay.io/hacbs-release/release-utils@sha256:53d4cdd2ff8d1a0c6d1781d754678b4c61e6fefa644c17bd0a4e2ca4336a3e18
+      script: |
+        #!/usr/bin/env sh
+        PATH=/bin:/usr/bin:/usr/local/bin
+        IR="ir-$(params.pipelineRunName)"
+        TASKRUN="/tmp/$$.sh"
+
+        cat > ${TASKRUN} <<SH
+        #!/usr/bin/env sh
+        while true; do
+          STATUS=`kubectl get internalrequest \${IR} -o jsonpath='{.status.conditions[0].status}'`
```

Comment:
Does the code break if the condition doesn't exist? Haven't tested myself.

Reply:
No, it does not.